database/schema/ve_db_dev_data.sql

```diff
@@ -158,4 +158,5 @@ SELECT refresh_all_cached_counts();
 SELECT setval('system_platform_id_seq', 100);
 SELECT setval('cve_metadata_id_seq', 100);
 SELECT setval('rh_account_id_seq', 100);
-SELECT setval('insights_rule_id_seq', 100);
\ No newline at end of file
+SELECT setval('insights_rule_id_seq', 100);
+SELECT setval('repo_id_seq', 100);
```

Comment:
This is here, because we have initial two repos as id 1 and 2 in the file, which created a conflict in tests with already existing PK.